.github/workflows/build.yml

```diff
@@ -46,7 +46,7 @@ jobs:
 
       - name: Installing dependencies
         run: |
-          curl -o compas.tar.gz -LJO https://pypi.debian.net/compas/latest
+          curl -o compas.tar.gz -LJO https://pypi.debian.net/compas/COMPAS-2.0.0a1.tar.gz
```

Comment:
I am just curious, why does it map dependencies to Compas-2.0.0a1 and not the latest?

Reply:
Recently a `2.0.0.a2` has been released where some Brep related stuff has been moved around, which breaks different things in `compas_timber`.
We already actually already had `2.0.0.a1` pinned in the [requirements.txt](https://github.com/gramaziokohler/compas_timber/blob/main/requirements.txt), but ironpython workflow uses this different mechanism to get it's dependencies and there it would just grab the latest.